app/src/main/java/org/oppia/android/app/preview/TopicPreviewActivity.kt

```diff
@@ -0,0 +1,52 @@
+package org.oppia.android.app.preview
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.android.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** TopicPreviewFragment TAG. */
+const val TOPIC_PREVIEW_FRAGMENT_TAG = "TopicPreviewFragment"
+private const val TOPIC_PREVIEW_ACTIVITY_TOPIC_ID_ARGUMENT_KEY = "TopicPreviewActivity.topic_id"
+private const val TOPIC_PREVIEW_ACTIVITY_PROFILE_ID_ARGUMENT_KEY = "TopicPreviewActivity.profile_id"
+
+/** Activity display preview of topic when topic is not downloaded. */
```

Comment:
```suggestion
/** Provides a preview of a topic for when it's not yet downloaded. */
```

Suggestion to make this a bit clearer.